style.css

```diff
@@ -0,0 +1,77 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background: #fff;
+  color: #1a2236;
+  margin: 0;
+  overflow: hidden;
+}
+
+a {
+  text-decoration: none;
+  background: #1a2236;
+  color: #fff;
+  padding: 12px 42px;
+}
+
+p {
+  font-family: Poppins, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  margin-top: 17px;
+  margin-bottom: 22px;
+}
+
+.primary-btn {
+  box-sizing: border-box;
+  background: #1a2236;
+  text-align: center;
+}
+
+.primary-btn:hover {
+  box-sizing: border-box;
+  background: #fff;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+}
+
+.primary-btn:active {
+  box-sizing: border-box;
+  background: #fff;
+  color: #1a2236;
+  border: 2px solid #1a2236;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+#full-name {
+  font-family: Playfair Display, sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-top: 30px;
+}
+
+.toggle {
+  padding: 19px 14px;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  padding: 0 14px 0 16px;
+}
```

Comment:
### The README file wasn't been added to this branch changes, so I'll just leave some comments related to the REAMDE file

1. Please consider deleting the link and putting a simple note like "Coming soon..." a clear notification for the readers that the link is not available currently 👍🏼 will just convey professionalism to the repo.
![image](https://user-images.githubusercontent.com/68030297/227176793-cd8ce768-db27-499a-b604-d01631fc6a39.png)

2. Readers with different backgrounds will be visiting this repo so I highly recommend that the getting started section should be descriptive and easy for non-technical readers to understand 💭 as an example explain how a non-technical person could clone your project.

![Screenshot from 2023-03-23 11-34-16](https://user-images.githubusercontent.com/68030297/227177169-8d684b71-78dc-4a26-8d90-17454c1451a4.png)